test/fixtures/postject-copy/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "postject-copy",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Optional nit: If this is code we're authoring and not copying from somewhere else, perhaps we should use the license that we use for the entire code base instead?

```suggestion
  "license": "MIT",
```